src/library/pagination/pagination.component.js

```diff
@@ -0,0 +1,170 @@
+// Import Template
+import template from './pagination.html';
+
+class controller {
+  constructor($log, $scope) {
+    Object.assign(this, {
+      $log,
+      $scope
+    });
+
+    this.currentPage = 0; // binding
+    this.disabled = false; // binding
+    this.itemsPerPage = 0; // binding
+    this.size = ''; // binding
+    this.totalItems = 0; // binding
+    this.visiblePageBuffer = 0; // binding
```

Comment:
I'll yank it unless we decide it's valuable to have across the board.  I value consistency almost above all. ;) It is sort of a pain to manage anyway and the bindings are right there at the bottom.